submit-web/src/components/registration/addProjects/ProjectCard/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const PROJECT_STATUS = {
+  APPROVAL: "Approved",
+  IN_REVIEW: "In Review",
```

Comment:
These are not related to the chip status on the right column. This is the work phase of the project
![image](https://github.com/user-attachments/assets/08b0bc23-ed8d-4cb3-9f89-8f6dfc96a72f)